roles/webots/localisation.role

```diff
@@ -19,7 +19,6 @@ nuclear_role(
   network::PlotJuggler
   # Platform
   input::SensorFilter
-  input::GameController
```

Comment:
This is needed for team robot localisation. I have a PR (#1583) that stops it from spamming when theres no game controller instance.